tests/NHSD.BuyingCatalogue.Solutions.API.UnitTests/SolutionsControllerGetPublicTests.cs

```diff
@@ -735,6 +736,40 @@ public async Task IfNativeDesktopThenNativeConnectivityDetailsIsSetCorrectly(str
             }
         }
 
+        [TestCase(null, null, false)]
+        [TestCase("", "     ", false)]
+        [TestCase("     ", "", false)]
+        [TestCase("     ", "        ", false)]
+        [TestCase("Component", null, true)]
+        [TestCase(null, "Capability", true)]
+        [TestCase("Component", "Capability", true)]
+
+        public async Task IfNativeDesktopEmptyThenThirdPartyHasNoData(string component, string capability, bool hasData)
```

Comment:
This name doesn't really describe what the test is testing.

Maybe something like:
NativeDesktopThirdPartySectionIsPopulatedCorrectly

?